src/compo/viirs_aod2ioda.py

```diff
@@ -99,6 +99,7 @@ def _read(self):
             vals = ncd.variables['AOD550'][:].ravel()
             errs = ncd.variables['Residual'][:].ravel()
             qcpath = ncd.variables['QCPath'][:].ravel()
+            qcpath = np.where(qcpath>-1,qcpath,np.nan)
```

Comment:
```suggestion
            qcpath = np.where(qcpath>-1, qcpath, np.nan)
```
I hope this suggestion makes it pass coding norms, may need to adjust the greater than too